packages/faceted-search/src/components/AddFacetPopover/AddFacetPopover.component.js

```diff
@@ -35,21 +33,16 @@ const isButtonDisabled = (badges, badgeDefinition, occurrences) => {
 const getTabIndex = isFocusable => (isFocusable ? 0 : -1);
 
 const OpenCategoryRow = ({ label, onClick, isFocusable }) => (
-	<TooltipTrigger label={label} tooltipPlacement="top">
-		<Button
-			aria-label={label}
-			bsStyle="link"
-			className={theme('tc-add-facet-popover-row')}
+	<div className={theme('tc-add-facet-popover-row')}>
+		<ButtonTertiary
```

Comment:
```suggestion
		<button
```
As we've discussed yesterday, this is probably your quickest bet for now

Reply:
Well, I managed to have something working well with Tertiary button now WDYT? As we know the long term solution/path should be to something close to the dropdown component from Coral